spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Dropout.scala

```diff
@@ -39,6 +39,7 @@ class Dropout(
   override private[mkldnn] def initFwdPrimitives(inputs: Array[MemoryData], phase: Phase) = {
     _inputFormats = inputs.map(x => HeapData(x.shape, format(x.shape)))
     _outputFormats = inputs.map(x => HeapData(x.shape, format(x.shape)))
+    _outputFormats.map(_.getPrimitive(runtime))
```

Comment:
why need this?

Reply:
Because the new reorder need them  to get the real/actual size of memory. We should create them here, so the `ReorderMemmory` can call `initActivity` or `initTensor` to get the real/actual native memory size.